### ISSUE TITLE
Tests fail on i586 architecture with pygit2-0.27.0

### ISSUE THREAD

**Issue by scarabeusiv** [libgit2/pygit2#795]
```[   99s] Traceback (most recent call last):
[   99s]   File "/home/abuild/rpmbuild/BUILD/pygit2-0.27.0/test/test_blame.py", line 67, in test_blame_index
[   99s]     self.assertEqualSignature(HUNKS[i][2], hunk.final_committer)
[   99s]   File "/home/abuild/rpmbuild/BUILD/pygit2-0.27.0/build/lib.linux-i686-2.7/pygit2/blame.py", line 73, in final_committer
[   99s]     return wrap_signature(self._hunk.final_signature)
[   99s]   File "/home/abuild/rpmbuild/BUILD/pygit2-0.27.0/build/lib.linux-i686-2.7/pygit2/blame.py", line 43, in wrap_signature
[   99s]     csig.when.time, csig.when.offset, 'utf-8')
[   99s] error: struct git_time: wrong total size (cdef says 12, but C compiler says 16). fix it or use "...;" in the cdef for struct git_time to make it flexible```

**Comment by jdavid** [comment]
Could not reproduce, but may be fixed by commit 1d11fa1
Can you verify? (and reopen if still fails)

**Comment by scarabeusiv** [comment]
Yep, the tests now pass, thanks.
